README.md

```diff
@@ -30,6 +30,7 @@ Unobtrusive, inheritable-overridable-configurable, drop-in debug logging, that w
 * *separate logger, if needed*
 * *log method calls, also when exit scope*
 * *Prevents heavy computation of strings with `logger.debug { 'log me' }` block format.*
+* *ActiveSupport::Notifications integration for instrumenting/logging events on class and instance methods*
```

Comment:
```suggestion
* *ActiveSupport::Notifications integration for instrumenting/logging events on class and instance methods*
* *Optional instance variable logging*
```

Reply:
Can you document the new support for instance variable logging also?

Reply:
Done 👍